pkg/config/protocol/instance.go

```diff
@@ -39,6 +39,8 @@ const (
 	HTTP2 Instance = "HTTP2"
 	// HTTPS declares that the port carries HTTPS traffic.
 	HTTPS Instance = "HTTPS"
+	// H2 declares that the port carries HTTP2 over TLS traffic.
+	H2 Instance = "H2"
```

Comment:
I worry about a user reasonably understanding the difference between `HTTP2` and `H2`. Its already confusing enough....

Reply:
Not to mention what if I want GRPC-over-tls, mongo-over-tls, etc

Reply:
Ya, it may be less confusing if another field adjacent to protocol is defined, e.g. `downgradedProtocol`. Only concern is if there are special alpns that I may not be aware of. And, for grpc it probably doesn't make sense to expose the http/1.1 alpn.